sample-java/build.gradle

```diff
@@ -24,10 +24,13 @@ android {
 dependencies {
   compile fileTree(dir: 'libs', include: ['*.jar'])
   compile project(':prado')
+  compile 'com.android.support:appcompat-v7:25.3.1'
+  compile 'com.android.support.constraint:constraint-layout:1.0.2'
+  compile 'com.squareup.picasso:picasso:2.5.2'
+  compile 'com.jakewharton.picasso:picasso2-okhttp3-downloader:1.1.0'
+  compile 'com.github.bumptech.glide:glide:3.8.0'
```

Comment:
Maybe we could have a couple flavors for the sample, with a different ImageLoader each? Just so it's clear that you only need one dependency or the other.